share/site/duckduckhack/inc/head.tx

```diff
@@ -1,3 +1,5 @@
+<title><: lp("title","DuckDuckHack - an open source community for DuckDuckGo Instant Answers") :></title>
```

Comment:
Is this the real verbiage? Isn't that too long for the tab? `DuckDuckHack - Instant Answers`?